features/request/request.feature

```diff
@@ -376,7 +376,7 @@ Feature: Formatting requests for sending to the ID Broker API
       And I provide a "value" of "01234987"
     When I call mfaVerify
     Then the method should be "POST"
-      And the url should be 'https://api.example.com/mfa/789/verify?rpOrigin=https%3A%2F%2Flogin.example.com'
+      And the url should be 'https://api.example.com/mfa/789/verify/?rpOrigin=https%3A%2F%2Flogin.example.com'
```

Comment:
You may need to also confirm that ID Broker handles calls to "/mfa/789/verify" (**without** trailing slash) the same way it handles calls to "/mfa/789/verify/" (**with** trailing slash). Those are technically two different routes, and we'll just want to make sure ID Broker handles them both the way this code is expecting it to.

Reply:
Oh, well. When I add a trailing slash in the idp-id-broker tests, they fail.

Reply:
I _think_ it would just be a matter of duplicating [this line](https://github.com/silinternational/idp-id-broker/blob/develop/application/frontend/config/main.php#L77) in idp-id-broker...

```php
'PUT     method/' . UID_ROUTE_PATTERN . '/verify' => 'method/verify',
```

... and giving the duplicate a trailing slash, so that Yii will route both of those URLs to that controller action:

```php
'PUT     method/' . UID_ROUTE_PATTERN . '/verify' => 'method/verify',
'PUT     method/' . UID_ROUTE_PATTERN . '/verify/' => 'method/verify',
```